script/build

```diff
@@ -5,18 +5,25 @@
 PLUGIN_NAME="${PLUGIN_NAME:?Must be set in .env}"
 PLUGIN_VERSION="${PLUGIN_VERSION:?Must be set in .env}"
 
+# This is necessary to get the project building on Go 1.6
+# Building the project will fail on Go 1.7+ as this env var will be ignored
+export GO15VENDOREXPERIMENT=0
```

Comment:
I will have a look into it. Let's leave the version support out of the picture for this PR and focus on the two issues it aims to address. We can track version support separately.